Update ble-cheat-sheet.md (#1810)
arduino/docs-content: content/hardware/03.nano/boards/nano-33-ble-sense/tutorials/cheat-sheet/ble-cheat-sheet.md

## RGB

> The Nano 33 BLE Sense's RGB LEDs are active-LOW, so they are switched in the opposite way to the LED_BUILTIN.

Reviewer: ```suggestion
The Nano 33 BLE Sense's RGB LED is active-LOW, so it is active when the pins are set to `LOW`.
```
